dot/network/config.go

```diff
@@ -52,6 +52,9 @@ const (
 
 	// DefaultDiscoveryInterval is the default interval for searching for DHT peers
 	DefaultDiscoveryInterval = time.Minute * 5
+
+	// defaultTxnBatchSize is the default size for the transaction batch
+	defaultTxnBatchSize = 100
```

Comment:
nit I know `tx` is widely used in the blockchain space, but let's perhaps use full words for the sake of readability / newcomers?

```suggestion
	defaultTransactionBatchSize = 100
```

Reply:
It will make the variable name very long which I am trying to avoid.

Reply:
Done